src/components/wizard-navigation-bar.component.horizontal.less

```diff
@@ -197,34 +217,33 @@ aw-wizard-navigation-bar.horizontal {
           .line(@big-dot-width, @big-dot-height, @wz-color-default);
         }
 
-        &:after {
+        .step-indicator {
           .state-circle(@big-dot-width, @big-dot-height, 0);
           .state-circle-with-background-and-content(@wz-color-default);
 
-          content: attr(step-symbol);
+          &:before {
+            content: attr(step-symbol);
```

Comment:
When we're already changing to a container instead of a selector for the navigation symbols, why do we need to hold on to `content: attr(step-symbol)`?

I mean wouldn't it be better to apply a similar strategy as for the step title and add `step-symbol` directly into the html template, instead of adding it via css?

Reply:
Makes sense, will do.

Reply:
fd5c402